frontend/src/metabase/sharing/components/SharingMenu/test/QuestionSharingMenu.enterprise.unit.spec.tsx

```diff
@@ -11,47 +11,37 @@ describe("QuestionSharingMenu > Enterprise", () => {
       canManageSubscriptions: true,
       isEmailSetup: true,
       isEnterprise: true,
-      isAdmin: false,
     });
     await openMenu();
     expect(screen.getByText("Create alert")).toBeInTheDocument();
   });
 
   describe("alerts permission disabled", () => {
-    it('Should not show the "Subscriptions" menu item to non-admins if the user lacks subscriptions/alerts permissions', async () => {
-      await setupQuestionSharingMenu({
+    it('should not show the "Subscriptions" menu item to non-admins if the user lacks alerts permissions', async () => {
+      setupQuestionSharingMenu({
         canManageSubscriptions: false,
         isEmailSetup: true,
         isEnterprise: true,
       });
       await openMenu();
-      expect(screen.queryByText("Create alert")).not.toBeInTheDocument();
-    });
-
-    it('should show a "Public link" button item if public sharing is enabled and the user lacks alerts permissions', async () => {
-      setupQuestionSharingMenu({
-        canManageSubscriptions: false,
-        isPublicSharingEnabled: true,
-        hasPublicLink: true,
-        isEnterprise: true,
-      });
-      const sharingButton = screen.getByTestId("sharing-menu-button");
 
-      expect(sharingButton).toBeEnabled();
-      expect(sharingButton).toHaveAttribute("aria-label", "Public link");
+      expect(screen.queryByText("Create alert")).not.toBeInTheDocument();
```

Comment:
all these modified tests will also assert that `create alert` isn't visible since this is under "alerts permission disabled"